SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/SpatialWorkerConnection.h

```diff
@@ -61,6 +61,8 @@ class SPATIALGDK_API USpatialWorkerConnection : public UObject, public FRunnable
 	FString GetWorkerId() const;
 	const TArray<FString>& GetWorkerAttributes() const;
 
+	void SetConnectionType(ESpatialConnectionType newType);
```

Comment:
We start with Capital letters :D 
Also I think this should just be `void SetConnectionType(ESpatialConnectionType ConnectionType);

Reply:
Done